docs/source/getting_started.rst

```diff
@@ -103,47 +162,130 @@ Arguments with default values can be annotated as follows:
            message += '!!!'
        return message
 
-Mixing dynamic and static typing within a single file is often
-useful. For example, if you are migrating existing Python code to
-static typing, it may be easiest to do this incrementally, such as by
-migrating a few functions at a time. Also, when prototyping a new
-feature, you may decide to first implement the relevant code using
-dynamic typing and only add type signatures later, when the code is
-more stable.
+``*args`` and ``**kwargs`` arguments can be annotated like so:
 
 .. code-block:: python
 
-   def f():
-       1 + 'x'  # No static type error (dynamically typed)
+   def stars(*names: str, **kwargs: float) -> float:
```

Comment:
I think that this example is a bit too clever. Maybe instead write too simpler (unrelated) functions, one for demonstrating `*args` and the other for `**kwargs`.